### PR TITLE
Remote Tools with APIv4 documentation

### PR DESCRIPTION
**Pull request by dontub**
Documentation for changes in #19.

systopia-reference: 23211

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,25 +3,93 @@
...
+civicrm_api4('RemoteMyEntity', 'get', ['profile' => 'my-profile']);
+```
+
+(Use the APIv4 explorer to test.)
+
+Most methods in the profile class have a parameter named `$contactId`. This
+contains the remote contact ID resolved to a CiviCRM contact ID or `NULL` if no
+remote contact ID was given. If a given remote contact ID could not be resolved,
+access is denied. The resolving is done using the extension
+[de.systopia.identitytracker](https://github.com/systopia/de.systopia.identitytracker).
```
Comment (jensschuppe) [inline README.md:71]: Is this suggesting that there's no support for anonymous API calls to any remote entity based on that structure?

File: README.md [hunk 2]
```diff
@@ -3,25 +3,93 @@
...
+
+You can find more information in the PHPDoc of the various classes, interfaces,
+and methods.
+
+All elements that can be safely used by third parties are marked with `@api`.
+There behavior will be stable within minor updates. Breaking changes may only
```
Comment (jensschuppe) [inline README.md:77]: `s/There/Their`

**Review by jensschuppe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dontub) [reply README.md:71]: Anonymous calls are possible:
> [...] resolved to [...] `NULL` if no remote contact ID was given.

Access is denied if a **given** remote contact ID could not be resolved.

Comment (jensschuppe) [reply README.md:71]: Thanks for clarifying, my bad!

File: README.md [hunk 3]
```diff
@@ -3,25 +3,93 @@
...
+  * [de.systopia.xcm](https://github.com/systopia/de.systopia.xcm)
+* Add the extension folder to your CiviCRM `ext` directory.
+* If your project uses composer: Add the requirements listed in `composer.json` to your composer project:
+  * [systopia/expression-language-ext](https://github.com/systopia/expression-language-ext)
+  * [systopia/opis-json-schema-ext](https://github.com/systopia/opis-json-schema-ext)
+  * [webmozart/assert](https://github.com/webmozarts/assert)
+* Otherwise: Run `composer update` inside the `de.systopia.remotetools` folder.
+* Enable the extension. (Either via UI or `cv ext:enable de.systopia.remotetools`)
+
+For the frontend part you can use the [civiremote Drupal module](https://github.com/systopia/civiremote).
```
[the remark this answers is not in the document]
Comment (dontub) [reply README.md]: Changed.

File: README.md (hunk 2 above)
Comment (dontub) [reply README.md]: Changed.

**Pull request merged**